.github/workflows/test.yml

```diff
@@ -31,6 +31,11 @@ jobs:
           NMSHD_TEST_BASEURL: http://localhost:8090
           NMSHD_TEST_CLIENTID: test
           NMSHD_TEST_CLIENTSECRET: test
+      - name: Upload coverage reports to Codecov with GitHub Action
+        uses: codecov/codecov-action@v4
+        env:
+          fail_ci_if_error: true
```

Comment:
I'm not a fan of this (in fact we should also disable this in the connector again). coverage should only be an information for us and not a hard requirement.

Reply:
That option has nothing to do with the coverage check, it is only for the upload of the coverage files.
We can disable this, but I think the CI check needs to be disabled in the codecov config.

Reply:
Ah, IC. then nvm. Upload errors SHOULD fail this ppl.